tests/unit/s2n_cipher_suite_match_test.c

```diff
@@ -156,35 +171,47 @@ int main(int argc, char **argv)
         /* Only two ciphers for testing RSA vs ECDSA. */
         uint8_t wire_ciphers_with_ecdsa[] = {
             TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,
-            TLS_RSA_WITH_AES_128_CBC_SHA256,
+            TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
         };
         const uint8_t cipher_count_ecdsa = sizeof(wire_ciphers_with_ecdsa) / S2N_TLS_CIPHER_SUITE_LEN;
 
-        EXPECT_NOT_NULL(server_config = s2n_config_new());
+        /* Only ECDSA ciphers */
+        uint8_t wire_ciphers_only_ecdsa[] = {
+            TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,
+            TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,
+            TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,
+            TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,
+            TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
```

Comment:
the comment and variable name says "only ecdsa" but this contains an ECDHE_RSA cipher suite

Reply:
Couldn't figure out why I added that. Removed it based on the comment in the test it's used in.